### PR TITLE
Bump NPM dependencies to their latest version

### PR DESCRIPTION
**Pull request by github-dd-nicolas**
## Proposed Release Notes
Bump NPM dependencies to their latest version

## Details

### CODE REVIEW

**Comment by bizob2828** [comment]
@github-dd-nicolas thanks for the Pr. This isn't necessary for you to get the functionality of 8.12.0. You simply just need to install newrelic@latest as it's a peer dep. I'll still take a look at it this a little later.

**Comment by github-dd-nicolas** [comment]
> @github-dd-nicolas thanks for the Pr. This isn't necessary for you to get the functionality of 8.12.0. You simply just need to install newrelic@latest as it's a peer dep. I'll still take a look at it this a little later.

Indeed, thanks a lot :+1:

File: package.json [hunk 1]
```diff
@@ -39,13 +39,13 @@
   "devDependencies": {
     "@newrelic/eslint-config": "^0.0.4",
     "@newrelic/newrelic-oss-cli": "^0.1.2",
-    "@newrelic/test-utilities": "^6.4.0",
-    "husky": "^7.0.4",
-    "lint-staged": "^12.3.3",
-    "newrelic": "^8.8.0",
-    "tap": "^16.0.1"
+    "@newrelic/test-utilities": "6.5.3",
```
Comment (bizob2828) [inline package.json:42]: we don't like to pin versions.  the best approach here would be to `npm i <pkg>@latest` which will upgrade it to the latest but keep the `^` matching

**Review by bizob2828 — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -435,10 +435,11 @@
       }
     },
     "node_modules/@grpc/proto-loader": {
-      "version": "0.6.9",
-      "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.6.9.tgz",
-      "integrity": "sha512-UlcCS8VbsU9d3XTXGiEVFonN7hXk+oMXZtoHHG2oSA1/GcDP1q6OUgs20PzHDGizzyi8ufGSUDlk3O2NyY7leg==",
+      "version": "0.6.12",
+      "resolved": "http://gv2-verdaccio-01.tools.exo.deindeal.ch:4873/@grpc%2fproto-loader/-/proto-loader-0.6.12.tgz",
```
Comment (bizob2828) [inline package-lock.json:439]: you must have some private npm proxy? please revert these. it's causing the build to fail and we also don't want to reference packages in some private mirror

Comment (github-dd-nicolas) [reply package-lock.json:439]: Sorry about it, I'll remove it

**Review by bizob2828 — APPROVED** [review]
thanks for keeping the deps up to date @github-dd-nicolas !

**Pull request merged**